mono/unit-tests/Makefile.am

```diff
@@ -129,10 +144,17 @@ check-local:
 	fi;
 
 clean-local:
-	rm -f callspec.exe callspec.pdb
+	rm -f callspec.exe callspec.pdb 
+	rm -f test-embed.exe test-embed.pdb
 
 %.exe: %.cs
 	$(MCS) -r:$(CLASS)/System.dll -r:$(CLASS)/System.Xml.dll -r:$(CLASS)/System.Core.dll -out:$@ $<
 
+build_mono_embed_test:
+	$(MCS) -r:$(CLASS)/System.dll -r:$(CLASS)/System.Xml.dll -r:$(CLASS)/System.Core.dll -out:test-embed.exe ../../samples/embed/test.cs 
```

Comment:
I think this can be a real make target:

```
test-embed.exe: $(top_srcdir)/samples/embed/test.cs
	$(MCS) -r:$(CLASS)/System.dll -r:$(CLASS)/System.Xml.dll -r:$(CLASS)/System.Core.dll -out:$@ $<
```

and 

```
test_mono_embed_DEPENDENCIES = test-embed.exe
```

If this doesn't work, please still use `$(top_srcdir)/samples/embed/test.cs` instead of `../../samples/embed/test.cs`.  That way we can do out of tree builds (some day...)

Reply:
Done